lib/safari/background.html

```diff
@@ -0,0 +1,21 @@
+{{#each background.scripts}}
+<script src="{{this}}">
+</script>
+{{/each}}
+<script>
+{{#if homepage_url}}
+// Default callback for homepageRedirect
+function homepageRedirec(){
+    safari.application.activeBrowserWindow.activeTab.url = "{{homepage_url}}";
```

Comment:
Can use default callback for redirect to homepage (also available for to be called from custom background scripts)